### ISSUE TITLE
Julia package behaves incorrectly with yay, works with pacman.

### ISSUE THREAD

**Issue by josswright** [Jguer/yay#789]
#### Affected Version
yay v8.1173.0 - libalpm v11.0.1

#### Issue
Package `julia` from the core repository works with `pacman` but fails with `yay`.
(`community/julia 2:1.0.1-2 `)

#### Output
Julia installs, apparently correctly, and runs when installed via `yay`. On running it, though, Julia's package manager does not pick up the default environment used internally for installing packages. This makes it effectively unusable.

Specifically, running `julia` in a terminal, then hitting `]` to enter the package manager produces a `pkg>` prompt, rather than selecting the default environment to produce a `(v1.0) pkg>` prompt. Trying to add packages using `add <PackageName>` will then fail with `ERROR: no active project`.

If julia is installed via `pacman`, Julia behaves correctly. There's some additional discussion [here](https://discourse.julialang.org/t/clean-julia-1-0-install-on-arch-linux-cant-add-packages/16931/8).

**Comment by Morganamilo** [comment]
Can't reproduce, nor can I imagine how installing it through yay would make it not work.

Try installing it through yay once again. If it is becomes broken again send the pacman log.

**Comment by josswright** [comment]
How frustrating. I now can't reproduce either. 

Uninstalling and reinstalling via yay now gives me a working Julia. Prior to this, with the non-working setup, I had uninstalled and reinstalled via yay several times without success. On reinstalling with pacman, on the suggestion of the other thread, the problem was resolved immediately. (You'll see on the Julia discourse page that I'd said myself I couldn't see how yay would be any different to pacman.)

Either way, as I now can't reproduce this either I suppose it can be closed. Hopefully, if anyone else runs into the same problem, this might magically work for them as well!
